linux_os/guide/services/ntp/chronyd_specify_remote_server/bash/shared.sh

```diff
@@ -4,6 +4,15 @@
 
 config_file="{{{ chrony_conf_path }}}"
 
+{{% if product in ["sle12", "sle15"] %}}
```

Comment:
@rumch-se  what is actually wrong with the remediation which is already in place? I tested it and it works well, it adds server entries. The remediation you added does not work unfortunately - no entries were added.